olive/engine/cloud_cache_helper.py

```diff
@@ -29,7 +29,6 @@ class CloudCacheConfig(ConfigBase):
 class CloudCacheHelper:
```

Comment:
While you are updating this could you also add some docstrings for the methods and comments about the cloud cache helpers? Otherwise, it's hard to follow the logic if this needs to be updated as part of other changes.

Reply:
Added